tests/src/test/resources/com/powsybl/openrao/tests/features/epic91_rao_enhancements/US91_13.feature

```diff
@@ -0,0 +1,193 @@
+    And the margin on cnec "cnecBeFr1Preventive" after PRA should be 579.65 A
+    And the tap of PstRangeAction "pstBeFr2" should be 7 after "Contingency BE1 FR1 3" at "curative"
+    And the value of the objective function after CRA should be 1382.77
+    And the worst margin is -1382.77 A
+    And the margin on cnec "cnecBeFr1Curative" after CRA should be -1382.77 A
+    And the margin on cnec "cnecBeFr2Curative" after CRA should be 15.49 A
+
+  @ac @fast @rao
+  Scenario: US 91.13.2.a: Unsecure case with a trade-off tap position - PST not is the limiting element
+  The curative scenario cannot be secured, the curative tap is chosen such that both curative CNECs are overloaded but
+  with the lowest global overload reachable. The PST is not the most limiting element at the end of curative.
+    Given network file is "epic91/2Nodes3ParallelLinesPST.uct"
+    Given crac file is "epic91/crac-91-13-2.json"
+    Given configuration file is "epic91/RaoParameters_ac.json"
+    When I launch search_tree_rao
+    Then the tap of PstRangeAction "pstBeFr2" should be -16 in preventive
+    And the margin on cnec "cnecBeFr1Preventive" after PRA should be 1479.65 A
+    And the tap of PstRangeAction "pstBeFr2" should be 4 after "Contingency BE1 FR1 3" at "curative"
+    And the value of the objective function after CRA should be 247.23
+    And the worst margin is -247.23 A
+    And the margin on cnec "cnecBeFr1Curative" after CRA should be -247.23 A
+    And the margin on cnec "cnecBeFr2Curative" after CRA should be -219.56 A
+
+  @ac @fast @rao @pst-regulation
+  Scenario: US 91.13.2.b: Duplicate of US 91.13.2.a with PST regulation
+  At the end of curative optimization, the PST is still unsecure but can be secured by moving the tap to position 7
+  even if it worsens the minimum margin significantly. Regulation cannot be performed because the PST is not the most
+  limiting element.
+    Given network file is "epic91/2Nodes3ParallelLinesPST.uct"
+    Given crac file is "epic91/crac-91-13-2.json"
+    Given configuration file is "epic91/RaoParameters_ac_pstRegulation.json"
+    When I launch search_tree_rao
+    Then the tap of PstRangeAction "pstBeFr2" should be -16 in preventive
+    And the margin on cnec "cnecBeFr1Preventive" after PRA should be 1479.65 A
+    And the tap of PstRangeAction "pstBeFr2" should be 4 after "Contingency BE1 FR1 3" at "curative"
+    And the value of the objective function after CRA should be 247.23
+    And the worst margin is -247.23 A
+    And the margin on cnec "cnecBeFr1Curative" after CRA should be -247.23 A
+    And the margin on cnec "cnecBeFr2Curative" after CRA should be -219.56 A
+
+  @ac @fast @rao
+  Scenario: US 91.13.3.a: 3 contingency scenarios and 3 PSTs with different use cases
+  - Scenario 12: PST 34 has a negative margin of -444.08 A so even though PST 12 should go up to -16 to maximize its own
+  margin, the minimal margin does not improve below -10 (margin at -9 is about -462 A)
```

Comment:
could you specify in the text: 'below tap -10'. i'm  a bit lost with the same vocabulary with the cnec pst and the range action pst